pkg/registry/client.go

```diff
@@ -209,7 +209,7 @@ func (c client) getAuthentication(ref name.Reference) remote.Option {
 		authn.DefaultKeychain,
 		authn.NewKeychainFromHelper(inlineHelper(c.config.GetExternalRegistryCredentials)),
 	)
-	if !env.LoadBooleanOrDefault(oktetoLocalRegistryStoreEnabledEnvVarKey, false) {
+	if !env.LoadBooleanOrDefault(oktetoLocalRegistryStorePriorityEnabledEnvVarKey, false) {
```

Comment:
Wasn't this flag supposed to not take into account local credentials? Did you have a bug in latest versions of CLI?

Reply:
We implemented it to not take in count the local credentials but during testing we decided to contemplate the local env vars and only change the priority based on the env var